app/models/journeys/early_years_payment/provider/authenticated/answers_presenter.rb

```diff
@@ -0,0 +1,81 @@
+module Journeys
+  module EarlyYearsPayment
+    module Provider
+      module Authenticated
+        class AnswersPresenter < BaseAnswersPresenter
+          include ActionView::Helpers::TranslationHelper
+
+          def claim_answers
+            [].tap do |a|
+              a << nursery
+              a << paye_reference
+              a << employee_name
+              a << start_date
+              a << child_facing_confirmation_given
+              a << returner
+              a << employee_email_address
+            end
+          end
+
+          private
+
+          def nursery
+            [
+              "Employee’s workplace",
+              EligibleEyProvider.find_by(urn: answers.nursery_urn).nursery_name,
+              "current-nursery"
+            ]
+          end
+
+          def paye_reference
+            [
+              "Employer’s PAYE reference number",
+              answers.paye_reference,
+              "paye-reference"
+            ]
+          end
+
+          def employee_name
+            [
+              "Employee's name",
```

Comment:
Small one, but there's a a few straight apostrophes in this file that need to be curly ’